mapdata/WarcraftLegacies/UnitData/Skin/1918984815.json

```diff
@@ -45,7 +45,7 @@
     {
       "Id": 1651864693,
       "Type": 3,
-      "Value": "Home to the warriors of the Horde.|n|cfff5962dTrains:|R Grunt, Headhunter, Bat Rider |n|cfff5962dResearches:|R Batriders, Airborne Toxins"
+      "Value": "Home to the warriors of the Horde.|n|cfff5962dTrains:|R Grunt, Headhunter, Catapult"
```

Comment:
This is wrong now, as War Camp still has those researches: 
![image](https://github.com/AzerothWarsLR/WarcraftLegacies/assets/8995690/1cd8dc72-95c7-47eb-8739-93652a6c127d)